pyspark-notebook/Dockerfile

```diff
@@ -17,7 +17,7 @@ USER root
 # (ARGS are in lower case to distinguish them from ENV)
 ARG spark_version="3.3.0"
 ARG hadoop_version="3"
-ARG scala_version="2.13"
+#ARG scala_version="2.13"
```

Comment:
By removing scala_version, are we defaulting to Scala 2.12 then?

Reply:
No and yes.. 
We will get the default Scala version. 
And in this case with Spark 3.3.0 we will get scala 2.12 
If you build it with --build-arg scala_version="2.13" when you will get Scala 2.13

Reply:
So the default build that gets published to the hub will be using 2.12 then? Is that what we want?

Reply:
That depends on whats are in the CI for building and push images to hub. 
With this PR users can chose between Scala versions, and the docker file will work as before.